client/src/ingredients/link-list.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function RenderLink({ link }) {
+  return (
+    <li>
+      <p>
+        <a href={link.mdn_url}>{link.short_title || link.title}</a>
```

Comment:
```suggestion
        <Link to={link.mdn_url}>{link.short_title || link.title}</Link>
```
you're also going to need to add `import { Link } from "@reach/router";`